src/app/frontend/common/components/resourcelist/event/template.html

```diff
@@ -62,14 +62,18 @@
 
       <ng-container [matColumnDef]="getDisplayColumns()[5]">
         <mat-header-cell *matHeaderCellDef>First Seen</mat-header-cell>
-        <mat-cell *matCellDef="let event">
+        <mat-cell *matCellDef="let event"
+          matTooltip="{{event.firstSeen | date:'medium'}}"
+          matTooltipPosition="above">
           {{event.firstSeen | date}}
         </mat-cell>
       </ng-container>
 
       <ng-container [matColumnDef]="getDisplayColumns()[6]">
         <mat-header-cell *matHeaderCellDef>Last Seen</mat-header-cell>
-        <mat-cell *matCellDef="let event">
+        <mat-cell *matCellDef="let event"
+          matTooltip="{{event.lastSeen | date:'medium'}}"
+          matTooltipPosition="above">
```

Comment:
We keep the default position in other places. Remove it and we can merge.

Reply:
@DanielUche Ping! Make that small change and we can merge this in. :) Thanks!